source/modules/task-list.js

```diff
@@ -1,15 +1,29 @@
+/** These variables are for the save task and edit task.
+ * Using the same form for save task and edit task
+ * SAVE_ON: adding new task
+ * EDIT_ON: eddting task
+ * TASK_CONTENT: the content of the current edited task
+ * if a task has more content, adjust this variable
+*/
+const SAVE_ON = true;
+const EDIT_ON = false;
+let SAVE_FLAG = SAVE_ON;
+let TASK_CONTENT = null;
+
+
 /**
  * @name addTaskButton
  * @function
  * @description Opens or closes add-task form
  */
-export function addTaskButton() {
+function addTaskButton() {
     let button = document.getElementById("add-task-form");
     if (button.classList.contains("hidden") === true) {
         button.classList.remove("hidden");
+        document.getElementById("task-name").focus();
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9414373)
```suggestion
        document.getElementById('task-name').focus();
```